<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CodeStyleStatisticsDialog</class>
 <widget class="QDialog" name="CodeStyleStatisticsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Code Style Checker Statistics</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTreeWidget" name="statisticsList">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>Count</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Code</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Message</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="filesChecked"/>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="filesIssues"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="totalIssues"/>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="ignoredIssues"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="fixedIssues"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>statisticsList</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CodeStyleStatisticsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CodeStyleStatisticsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
